site/public/css/navigation.css

```diff
@@ -62,21 +71,82 @@
     display: none;
 }
 
-@media (min-width: 541px) {   
+.course-section-title {
+    display: inline;
+}
+
+@media (max-width: 401px) {
     .course-section-heading {
-        border-bottom: 2px solid #000;
-        background: none;
+        font-size: 14px;
     }
+    .course-main .course-title {
+        font-size: 12px;
+    }
+    .course-button .btn, .subtitle {
+        font-size: 11px;
+    }
+}
 
-    .gradeable-row {
-        border: none;
```

Comment:
Personally I think the borders on desktop are a bit weird looking. This css is the line that used to hide them. I think they look fine on mobile if you want to leave them. My issue with them is that it makes it look like each item is its own section instead of part of another section (ex:  BETA,  OPEN, CLOSED, etc.). This might be because it stretches the full width even though the lines for each section stop with a bit of space on the left and right side.

This is what your code currently looks like 
![image](https://user-images.githubusercontent.com/18558130/82608584-ff577280-9b88-11ea-912f-5f0461f9c4de.png)
I think it should look like this (which is how it used to look)
![image](https://user-images.githubusercontent.com/18558130/82608667-19915080-9b89-11ea-9788-f626c7cd7370.png)

The mobile version I think looks fine with or without the bottom border line.

Reply:
removed border from the gradeables when shown in row-form.